partitura/utils/music.py

```diff
@@ -915,10 +929,15 @@ def estimate_symbolic_duration(dur, div, eps=10**-3):
         Number of units per quarter note
     eps : float, optional (default: 10**-3)
         Tolerance in case of imprecise matches
+    return_com_durations : bool, optional (default: False)
+        If True, return composite durations as well.
 
     Returns
     -------
-
+    out: Union[Union[Dict[str, Any], Tuple[Dict[str, Any]]], None]
+        Symbolic duration as a dictionary, or None if no matching
+        duration is found. When a composite duration is found, then it returns a tuple of symbolic durations.
+        The returned tuple should be tied notes.
```

Comment:
"the returned tuple contains tied notes"

Reply:
I am not sure I understand your comment. In the case of composite durations, the function returns a tuple of symbolic durations, these durations should correspond to many notes tied together. It does not return note objects.

Reply:
Addressed with commit [61881a48](https://github.com/CPJKU/partitura/pull/352/commits/61881a482f6cc70fd2184f8a6546f22c0afd2172)